src/components/GiveProject/GiveHeader.tsx

```diff
@@ -27,23 +27,23 @@ export function GiveHeader({ isSmallScreen, isVerySmallScreen, totalDebt, networ
         to="/give"
         className={`give-option ${isGiveTabActive ? "give-active" : ""}`}
       >
-        <Button color="secondary">
+        <SecondaryButton>
           <Typography variant="h6">
             <Trans>Give</Trans>
           </Typography>
-        </Button>
+        </SecondaryButton>
       </Link>
       <Link
         component={NavLink}
         id="give-sub-donations"
         to="/give/donations"
         className={`give-option ${isDonationsTabActive ? "give-active" : ""}`}
       >
-        <Button color="secondary">
+        <SecondaryButton>
           <Typography variant="h6">
```

Comment:
most buttons do not need to be wrapped in typography tags unless it significantly alters the style of the button